db/seeds.rb

```diff
@@ -13,11 +13,18 @@
 puts 'Admin credentials: %s' % [admin.password]
 
 if File.exist?([Rails.root, '/config', '/seed.yml'].join)
-  seed_file = File.join(Rails.root, 'config', 'seed.yml')
-  seed = YAML.load(File.open(seed_file))
+  seed_file   = File.join(Rails.root, 'config', 'seed.yml')
+  seed        = YAML.load(File.open(seed_file))
+
+  app_seed    = seed['oauth_application']
+  application = Doorkeeper::Application.new(skipauth: app_seed['skipauth'], name: app_seed['oauth_app_name'], redirect_uri: app_seed['oauth_callback_url'])
+
+  unless ENV['WORKBENCH'].nil?
```

Comment:
Let's do it like this: `if ENV['WORKBENCH'].in?(%w[ 1 true ])`. Also we don't need `seed.yml` in this case so check if it's the workbench before.

Reply:
this need to move in yml files seed.yml that will be mounted.